client/blocks/product-purchase-features-list/index.jsx

```diff
@@ -167,9 +185,13 @@ class ProductPurchaseFeaturesList extends Component {
 	}
 
 	getJetpackPersonalFeatures() {
-		const { selectedSite } = this.props;
+		const { isAutomatedTransfer, isPlaceholder, selectedSite } = this.props;
 
 		return [
+			<HappinessSupportCard
+				isJetpack={ !! selectedSite.jetpack && ! isAutomatedTransfer }
+				isPlaceholder={ isPlaceholder }
```

Comment:
We don't seem to take the `isPlaceholder` prop in consideration in `<HappinessSupportCard />`.